tests/integration/concrete/campaign/lt/constructor.t.sol

```diff
@@ -42,7 +42,7 @@ contract Constructor_MerkleLT_Integration_Test is Integration_Test {
 
     function test_Constructor() external {
         // Make Factory the caller for the constructor test.
-        resetPrank(address(merkleFactory));
+        resetPrank(address(merkleFactoryBase));
```

Comment:
Should be `merkleFactoryInstant` here and same in other constructor contracts in merkle campaign directory.